libs/deso-protocol/src/lib/internal.ts

```diff
@@ -38,21 +53,192 @@ export const handleSignAndSubmit = async (
   params: OptionalFeesAndExtraData & any,
   // we always broadcast by default, but consumers can optionally disable it.
   options: RequestOptions = { broadcast: true }
-) => {
-  const constructedTransactionResponse = await api.post(
-    options.nodeURI ? `${cleanURL(options.nodeURI, endpoint)}` : endpoint,
-    {
-      ...params,
-      MinFeeRateNanosPerKB:
-        params.MinFeeRateNanosPerKB ?? globalConfigOptions.MinFeeRateNanosPerKB,
-    }
-  );
-  const submittedTransactionResponse = options.broadcast
-    ? await identity.signAndSubmit(constructedTransactionResponse)
-    : null;
+): Promise<{
+  constructedTransactionResponse: ConstructedTransactionResponse;
+  submittedTransactionResponse: SubmitTransactionResponse | null;
+}> => {
+  const constructedTransactionResponse = await ((options.localConstruction ||
+    globalConfigOptions.LocalConstruction) &&
+  options.constructionFunction
```

Comment:
are there cases where we would pass the `localConstruction` option (or set the global flag) and not have a local constructionFunction? It seems like maybe that should be an error scenario?

Reply:
There are certain functions, like `UpdateProfile` or `BuyCreatorCoin` that are not feasible to construct locally. In this case, I thought it'd be an easier dev experience (at least internally) for testing to just get `globalConfigOptions.LocalConstruction` to true instead of updating each function individually.